siren-ai.md

```diff
@@ -0,0 +1,17 @@
+---
+permalink: /siren-ai/
+---
+# Siren AI Plugin
+The Siren AI plugin integrates Investigate with various LLM providers enabling users to leverage large language models (LLM) within their Investigate environment.
+
+This plugin offers the following features:
+
+* Generate comprehensive reports by analyzing user-created graphs.
+
+* Extends the Siren API scripting environment by providing a set of methods for interacting programmatically with a configured LLM. This allows for the customization of UIs and workflows that integrate generative AI.
+
+The supported LLM providers are OpenAI, Azure OpenAI, Ollama, and AWS Bedrock.
+
```

Comment:
Can you add the install instructions or at least a link to the page on the docs?

Reply:
Crap, yeah, will do!

Reply:
Updated, the docs version will become `1.0` instead of `1.0.0`. This change is being made here https://github.com/sirensolutions/siren-ai-plugin/pull/42

Reply:
This is ok for now but you'll have to update it everytime you do a release, which is a pain. 

There is _some_ way of setting a latest flag in antora to redirect to the newest version but I've forgotten what it is and I can't find the 2.x docs online anymore. :(

If you can set that in the antora.yml for the docs, you could point this at docs.siren.io/latest/siren-ai/t_installing.html and it would redirect.

Reply:
>This page in the 3.x antora docs explains the idea. This might work, I'm not sure.
  https://docs.antora.org/antora/latest/playbook/urls-latest-version-segment/

Hmm, I don't think so, that seems to apply a latest version globally instead of just a single source e.g `siren-ai`

Reply:
From reading the docs there, it should work per source (or "component" as they call them in the antora docs).